sdk/fluidrelay/azure-resourcemanager-fluidrelay/src/test/java/com/azure/resourcemanager/fluidrelay/FluidRelayManagerTests.java

```diff
@@ -0,0 +1,103 @@
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.fluidrelay.models.FluidRelayServer;
+import com.azure.resourcemanager.fluidrelay.models.Identity;
+import com.azure.resourcemanager.fluidrelay.models.ProvisioningState;
+import com.azure.resourcemanager.fluidrelay.models.ResourceIdentityType;
+import com.azure.resourcemanager.fluidrelay.models.StorageSku;
+import com.azure.resourcemanager.resources.ResourceManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class FluidRelayManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST2;
+    private String resourceGroupName = "rg" + randomPadding();
+    private FluidRelayManager fluidRelayManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        fluidRelayManager = FluidRelayManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void testCreateFluidRelayServer() {
+        FluidRelayServer server = null;
+        try {
+            String serverName = "server" + randomPadding();
+            // @embedmeStart
+            server = fluidRelayManager.fluidRelayServers()
+                .define(serverName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withIdentity(new Identity().withType(ResourceIdentityType.NONE))
+                .withStoragesku(StorageSku.STANDARD)
+                .withProvisioningState(ProvisioningState.SUCCEEDED)
```

Comment:
I don't think user need to set this.

Reply:
Only the necessary basic settings have been retained in the new version.